### PR TITLE
Update changelog for v3.0.1

### LINKED ISSUE #492: Update Changelogs for 3.0.1
**Original report on BitBucket by John Shepherdson (GitHub: [john-shepherdson](https://github.com/john-shepherdson)).**

----------------------------------------

Ensure CHANGELOG and Release report are aligned.

See [https://docs.google.com/document/d/1O1P3bT9I2P\_LRrLtnyXCNPl05URPc0\_LyaddKhR5Gn0/edit#](https://docs.google.com/document/d/1O1P3bT9I2P_LRrLtnyXCNPl05URPc0_LyaddKhR5Gn0/edit#)

### PR DESCRIPTION
**Pull request by matthew-morris-cessda**
This closes #492.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,25 +14,26 @@ _For each release, use the following sub-sections:_
 - _Fixed (for any bug fixes)_
 - _Security (in case of vulnerabilities)_
 
-## [3.1.0] - UNRELEASED
+## [3.0.1] - 2023-03-07
 
 - [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.xxxx.svg)](https://doi.org/10.5281/zenodo.xxxx)
```
Comment (john-shepherdson) [inline CHANGELOG.md:19]: Remove this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,25 +14,26 @@ _For each release, use the following sub-sections:_
...
 
 ### Additions
 
-- [#xxx](https://github.com/cessda/cessda.cvs.two/issues/xxx) DESCRIPTION
+- [#482](https://github.com/cessda/cessda.cvs.two/issues/482) Add the Maven Release Plugin to the POM
 
-### Changes
+### Fixes
```
Comment (john-shepherdson) [inline CHANGELOG.md:25]: What about 554, 491, 489, 464?

File: CHANGELOG.md (hunk 1 above)
Comment (john-shepherdson) [reply CHANGELOG.md:19]: as there will be no DOI issued for this version

File: CHANGELOG.md (hunk 2 above)
Comment (matthew-morris-cessda) [reply CHANGELOG.md:27]: Changelog entries have been added

**Pull request merged**